api/src/main/resources/org/openmrs/module/mirebalaisreports/sql/mexicoCesMeds.sql

```diff
@@ -16,8 +17,10 @@ FROM obs disp_quant
                             AND crs.name LIKE "PIH"
                       )
                   )
-         INNER JOIN drug d
-                    ON disp_drug.value_drug = d.drug_id
+         JOIN drug d
```

Comment:
Did you intend to change this from INNER JOIN to JOIN? I forget if these are the same, but it would be better to be explicit.

Reply:
Yeah, they're the same. Plain JOIN is used elsewhere in the file. Good to know explicitly naming the join is preferred.